app/models/outcome_with_metadata.rb

```diff
@@ -1,3 +1,7 @@
 class OutcomeWithMetadata < Outcome
   self.table_name = "outcomes_with_metadata"
+
+  def nickname
```

Comment:
Is this needed? `OutcomeWithMetadata` is a view that already selects `outcomes.nickname`. So theoretically, `nickname` should already be a method on this class.